src/lib/data-collector.js

```diff
@@ -43,10 +43,14 @@ DataCollector.prototype.log = function (message) {
 
 DataCollector.prototype.getDeviceData = function () {
   if (!this._instance) {
-    return '';
+    return Promise.resolve('');
   }
 
-  return this._instance.deviceData;
+  // TODO do we want to maintain the behavior
```

Comment:
Not yet, I'll ask the API Support folks.

Reply:
IMO, we should error out if data collector fails to set up. In almost all cases, it'll fail because the merchant specified `kount` in their config, but they were not enabled for Kount.